mllib/src/main/scala/org/apache/spark/ml/classification/OneVsRest.scala

```diff
@@ -209,4 +215,12 @@ final class OneVsRest(override val uid: String)
     val model = new OneVsRestModel(uid, labelAttribute.toMetadata(), models).setParent(this)
     copyValues(model)
   }
+
+  override def copy(extra: ParamMap): OneVsRest = {
+    val copied = super.copy(extra).asInstanceOf[OneVsRest]
+    if (isDefined(classifier)) {
+      copied.setClassifier($(classifier).copy(extra))
```

Comment:
When writing copy for CrossValidator, I recall we decided not to copy the extra params to the estimator param.
This case seems analogous, so we probably should not copy the extra params to classifier here.
(I could imagine arguments for either behavior, but we should be consistent.)